.github/workflows/pull_request.yaml

```
@@ -0,0 +1,41 @@
+name: Build Chain
+
+on: [pull_request]
+
+jobs:
+  build-chain:
+    concurrency:
+      group: pull_request-${{ github.head_ref }}
+      cancel-in-progress: true
+    strategy:
+      matrix:
+        java-version: [8, 11]
+      fail-fast: true
+    runs-on: ubuntu-latest
+    name: Maven Build
+    steps:
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java-version }}
+      # See https://docs.github.com/en/actions/guides/building-and-testing-java-with-maven#caching-dependencies
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-${{ matrix.java-version }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-${{ matrix.java-version }}-m2
+      - name: Build Chain ${{ matrix.java-version }}
+        id: build-chain
+        uses: kiegroup/github-action-build-chain@v2.6.4
+        with:
+          definition-file: https://raw.githubusercontent.com/${GROUP}/${PROJECT_NAME}/${BRANCH}/.ci/pull-request-config.yaml
```

sorry, I guess this is taken from https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/.github/workflows/pull_request.yml but it should be
```suggestion
          definition-file: https://raw.githubusercontent.com/${GROUP}/droolsjbpm-build-bootstrap/${BRANCH}/.ci/pull-request-config.yaml
```

---

DONE